apps/admin-server/src/pages/projects/[project]/authentication/requiredfields.tsx

```diff
@@ -205,12 +214,53 @@ export default function ProjectAuthenticationRequiredFields() {
                 )}
               />
 
+              {showPageFields ? (
+                  <>
+                    <Spacer size={2} />
+                    <div>
+                      <FormLabel>
+                        Standaard staat de titel van de bovenstaande geselecteerde verplichte velden als de titel boven het invulveld. Hier kun je dit per verplicht veld aanpassen.
+                      </FormLabel>
+                    </div>
+                    <div className="grid grid-cols-1 lg:grid-cols-2 gap-4">
+                      {requiredUserFields.map((item) => {
+                        const fieldValue = form.getValues('requiredUserFieldsLabels')[item.id] ?? '';
+                          return form.watch('requiredUserFields').includes(item.id) ? (
+                              <FormField
+                                  key={item.id}
+                                  control={form.control}
+                                  name={`requiredUserFieldsLabels.${item.id}`}
+                                  render={({field}) => (
+                                      <FormItem>
+                                        <FormLabel>{item.label}</FormLabel>
+                                        <FormControl>
+                                          <Input
+                                              placeholder={item.label}
+                                              defaultValue={fieldValue}
+                                              onChange={(e) => {
+                                                field.onChange(e);
+                                              }}
+                                          />
+
+                                        </FormControl>
+                                        <FormMessage/>
+                                      </FormItem>
+                                  )}
+                              />
+                          ) : null
+                      })}
+                    </div>
+                    <Spacer size={5} />
+                  </>
+              ) : null}
+
+
               {showPageFields ? (
               <>
               <Separator className="my-4" />
               <div>
                 <FormLabel>
-                  Als een gebruiker één of meer ven deze verplichte velden moet invullen dan doet die dat op een pagina met deze teksten:
+                  Als een gebruiker één of meer een deze verplichte velden moet invullen dan doet die dat op een pagina met deze teksten:
```

Comment:
Denk dat de tekst nog niet helemaal klopt: Als een gebruiker één of meer `van` deze verplichte velden moet invullen dan doet die dat op een pagina met deze teksten: